ports/litex/supervisor/port.c

```diff
@@ -84,7 +84,7 @@ void reset_port(void) {
 }
 
 void reset_to_bootloader(void) {
-    reboot_ctrl_write(0xac);
+    //reboot_ctrl_write(0xac);
```

Comment:
Will this break Fomu?

Reply:
Yes, I'll fix that.

It will be possible to simply reset the board, but with the current bootloader I'm not sure we can perform a "reset into the bootloader", the multi-boot features are different on ECP5 as they are on the ice40. In it's current form the bootloader is watching for the button to be pressed.

I'll need to think how we can handle this, it may require using volatile bits in a FLASH config register.

Reply:
You could also reserve a part of the DDR for it. We've had a similar problem with the ESP32-S2 which only has a power enable line and not a reset line and therefore blanks all ram on external "reset".